configs/tridentnet/README.md

```diff
@@ -2,6 +2,8 @@
 
 ## Introduction
 
+Please note that similar to [Detectron2](https://github.com/facebookresearch/detectron2/tree/master/projects/TridentNet), we haven't implemented the Scale-aware Training Scheme in section 4.2 of the paper.
```

Comment:
Put the note in the results and models section after the table.
The note should start with `**Note**`.